dbfit-java/build.gradle

```diff
@@ -293,6 +386,40 @@ project('teradata') {
         runtime 'com.teradata:terajdbc4:15.0.0.15@jar'
         runtime 'com.teradata:tdgssconfig:15.0.0.15@jar'
     }
+
+    ext.connectionPropertiesFiles = ['TestDbConnectionDbFitTeradata.properties']
+    ext.adminConnectionPropertiesFile = 'TestDbConnectionTeradataAdmin.properties'
+
+    task deploymentProperties.doLast {
+        ext {
+            jdbcDriverClass = 'com.teradata.jdbc.TeraDriver'
+            jdbcDriverUrlPrefix = 'jdbc:teradata://'
+            delimiterType = 'row'
+            autoCommit = true
+        }
+    }
+
+    dbConnectionProperties.dependsOn(connectionFileProperties)
+
+    def getJdbcUrl = {
+        service, database ->
+            return deploymentProperties.jdbcDriverUrlPrefix + service +
+                '/TMODE=TERA,DATABASE=' + database
+    }
+
+    dbConnectionProperties.doLast {
+        ext {
+            jdbcUrl = getJdbcUrl(connectionFileProperties.service, connectionFileProperties.database)
```

Comment:
When setting just a single property it's a bit more concise to have just `ext.jdbcUrl`.